src/configure/activate.ts

```diff
@@ -27,15 +22,12 @@ export async function activateConfigurePipeline(context: vscode.ExtensionContext
     // The command has been defined in the package.json file
     // Now provide the implementation of the command with registerCommand
     // The commandId parameter must match the command field in package.json
-    vscode.commands.registerCommand('configure-pipeline', async (node: any) => {
+    registerCommand('configure-pipeline', async (actionContext: IActionContext, node: any) => {
```

Comment:
is actionContext always a new fresh object? or is it shared amongst all invocations of this particular command?

Reply:
it is a new fresh object each time